website/status.php

```diff
@@ -1,37 +1,55 @@
 <?php
+
 @define('CONST_ConnectionBucket_PageType', 'Status');
 
 require_once(dirname(dirname(__FILE__)).'/settings/settings.php');
 require_once(CONST_BasePath.'/lib/init-website.php');
+require_once(CONST_BasePath.'/lib/ParameterParser.php');
+require_once(CONST_BasePath.'/lib/Status.php');
 
-
-function statusError($sMsg)
-{
-    header('HTTP/1.0 500 Internal Server Error');
-    echo 'ERROR: '.$sMsg;
-    exit;
-}
+$oParams = new Nominatim\ParameterParser();
+$sOutputFormat = $oParams->getSet('format', array('text', 'json'), 'text');
+$bForceError = $oParams->getInt('force_error', false);
```

Comment:
Was this for debugging? Can this go away now?

Reply:
The `force_error` parameter is for BDD tests (test/bdd/api/status/simple.feature) because the /status endpoint for text output returns HTTP 500, but for JSON returns HTTP 200.

Reply:
I'm reluctant to accept test feature like that in a production API. If you want to test the error condition, how about sneaking a bad database name into the test? You could add a context tag that forces a different DB name. Have a look at before_scenario() in tests/bdd/environment.py. That might be a good place to start.